### PR TITLE
docs(text-input): add light themed password input example

### LINKED ISSUE #1137: text-input--toggle-password-visibility is missing light version & codepen
Hi @emyarod ! Went to start adding this to the new website and realized we need the light version defined in the handlebar template and config in order to display both versions on the website.

### PR DESCRIPTION
**Pull request by emyarod**
Closes #1137

#### Changelog

**Changed**

- include light themed password input section in documentation

### CODE REVIEW

File: src/components/text-input/text-input.config.js [hunk 1]
```diff
@@ -20,10 +20,18 @@ module.exports = {
     },
     {
       name: 'toggle password visibility',
-      label: 'Text Input (toggle password visibility)',
+      label: 'Password Input (toggle password visibility)',
```
Comment (alisonjoseph) [inline src/components/text-input/text-input.config.js]: I think we could just leave this as `label: 'Password Input'` We don't add additional descriptors to any of the other components. What do you think?

**Review by alisonjoseph — APPROVED** [review]
LGTM other than one comment/question. 😄

**Comment by alisonjoseph** [comment]
One more question, does the icon shift up one pixel when you click it? Is that just a misalignment in the two icons?

**Comment by emyarod** [comment]
image
 
@alisonjoseph you're right, the height attribute of the `icon--visibility-on` SVG is 11px whereas `icon--visibility-off` height is 16px. Right now I'm not applying any styles to the icons.

 Do we want to standardize the icon dimensions in the `carbon-icons` source SVGs, or should we modify the `carbon-components` SCSS?

File: src/components/text-input/text-input.config.js (hunk 1 above)
Comment (emyarod) [reply src/components/text-input/text-input.config.js]: yeah that's probably better @alisonjoseph, I've changed it

**Comment by alisonjoseph** [comment]
Are both icons being pulled from carbon-icons? If so we should probably fix there. Or fix the icon in this PR and then get it added back in carbon-icons.

**Comment by emyarod** [comment]
They are hard coded in the handlebars template, but the paths are taken straight from `carbon-icons`. Somewhat related was one of the questions from #1060 about being able to import the SVGs from the `carbon-icons` package rather than hard coding them

**Comment by alisonjoseph** [comment]
Right, I just meant let's update the svg and make sure it gets updated here and in carbon-icons.

**Comment by emyarod** [comment]
it should be addressed in https://github.com/IBM/carbon-components/pull/1145/commits/443043b7dcd00db1acc72e5fe7a7a384b542af74 and https://github.com/IBM/carbon-icons/pull/84 now

**Comment by alisonjoseph** [comment]
Awesome thanks!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @emyarod!

**Pull request merged**